test/unit/workers/document_list_export_worker_test.rb

```diff
@@ -57,19 +57,6 @@ class DocumentListExportWorkerTest < ActiveSupport::TestCase
     assert_equal "this|is|my|csv", file.body
   end
 
-  test "email body includes url" do
```

Comment:
The "Fixes tests" commit should be fixed up to the first commit.

Reply:
Yep — wanted to keep it around temporarily because I wasn't sure about removing one of the tests, but I won't merge as-is.